Make.config

```diff
@@ -311,7 +311,7 @@ EXTRA_SIMULATORS=com.apple.pkg.iPhoneSimulatorSDK14_3 com.apple.pkg.AppleTVSimul
 
 INCLUDE_IOS=1
 INCLUDE_MAC=1
-INCLUDE_WATCH=1
+INCLUDE_WATCH=
 INCLUDE_TVOS=1
```

Comment:
Does the code properly handle it when some platforms are disabled - for instance if you do:

```make
INCLUDE_TVOS=
```

Reply:
You can test this by make the change in the PR to check if everything works correctly, so three important variations:

1. Disable any specific platform (set `INCLUDE_TVOS=` for instance)
2. Disable .NET (set `ENABLE_DOTNET=`)
3. Disable legacy Xamarin (set `INCLUDE_XAMARIN_LEGACY=`)

So that would be three separate commits (to speed this up you could also create three new branches branched off from this branch, with these changes, and create a PR for each with the target branch being this branch).